pkg/sac/allow_fixed_scopes_checker_core.go

```diff
@@ -12,8 +12,10 @@ type allowFixedScopesCheckerCore []scopeKeySet
 // AllowFixedScopes returns a scope checker core that allows those scopes that
 // are in the cross product of all individual scope key lists. I.e.,
 // AllowFixedScopes(
-//   AccessModeScopeKeys(storage.Access_READ, storage.Access_READ_WRITE),
-//   ResourceScopeKeys(resources.CLUSTER),
+//
+//	AccessModeScopeKeys(storage.Access_READ, storage.Access_READ_WRITE),
+//	ResourceScopeKeys(resources.CLUSTER),
+//
```

Comment:
This has wrong indentation compare with master https://github.com/stackrox/stackrox/blob/master/pkg/sac/allow_fixed_scopes_checker_core.go#L15-L18